projects/2024/neutron-ball/report-analytical.ipynb

```diff
@@ -179,13 +180,13 @@
     "\n",
     "\\begin{align*}\n",
     "y = c_1 e^{m_1x} + c_2 e^{m_2 x} \n",
-    "    + \\frac{e^{m_1x}}{m_1-m_2} \\int R\\left(x\\right) e^{-m_1x} \\, dx\n",
```

Comment:
The integrand and the differential should be separated by a space: $\int x dx$ and $\int x\\,dx$.